tanjun/abc.py

```diff
@@ -362,6 +423,36 @@ async def edit_initial_response(
             typing.Union[hikari.SnowflakeishSequence[hikari.PartialRole], bool]
         ] = hikari.UNDEFINED,
     ) -> hikari.Message:
+        """Edit the initial response for this context.
+
+        Parameters
+        ----------
+        content : hikari.UndefinedOr[typing.Any]
+            The content to edit the response with.
```

Comment:
Should probably document what's accepted here (the api.rest.RESTClient's docs may be good inspiration for this)